packages/components/src/tooltip/test/index.js

```diff
@@ -1,193 +1,165 @@
 }
 
 describe( 'Tooltip', () => {
 	describe( '#render()', () => {
-		it( 'should not render the tooltip if multiple children are passed', () => {
+		it( 'should not render the tooltip if multiple children are passed', async () => {
+			const user = userEvent.setup();
+
 			render(
-				<Tooltip text="Help text">
-					<button>Button 1</button>
-					<button>Button 2</button>
+				<Tooltip { ...props }>
+					<Button>Button 1</Button>
+					<Button>Button 2</Button>
 				</Tooltip>
 			);
 
-			const button = screen.getByText( 'Button 1' );
-			act( () => button.focus() );
-			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
+			await user.tab();
+
+			expect(
+				screen.queryByText( 'tooltip text' )
+			).not.toBeInTheDocument();
 		} );
 
-		it( 'should render children', () => {
+		it( 'should not render the tooltip if there is no focus', () => {
 			render(
-				<Tooltip text="Help text">
-					<button>Hover Me!</button>
+				<Tooltip { ...props }>
+					<Button>Hover Me!</Button>
 				</Tooltip>
 			);
 
 			expect(
 				screen.getByRole( 'button', { name: 'Hover Me!' } )
 			).toBeVisible();
-			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
+			expect(
+				screen.queryByText( 'tooltip text' )
+			).not.toBeInTheDocument();
 		} );
 
-		it( 'should render children with additional tooltip when focused', async () => {
-			const mockOnFocus = jest.fn();
+		it( 'should render the tooltip when focusing anchor via tab', async () => {
+			const user = userEvent.setup();
 
 			render(
-				<Tooltip text="Help text">
-					<button onFocus={ mockOnFocus }>Hover Me!</button>
+				<Tooltip { ...props }>
+					<Button>Hover Me!</Button>
 				</Tooltip>
 			);
 
-			const button = screen.getByRole( 'button', { name: 'Hover Me!' } );
-			expect( button ).toBeVisible();
-
-			// Before focus, the tooltip is not shown.
-			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
+			await user.tab();
 
-			act( () => button.focus() );
+			expect(
+				screen.getByRole( 'button', { name: /Hover me!/i } )
+			).toHaveFocus();
 
-			// Tooltip is shown after focusing the anchor.
-			const tooltip = screen.getByText( 'Help text' );
-			expect( tooltip ).toBeVisible();
-			expect( mockOnFocus ).toHaveBeenCalledWith(
-				expect.objectContaining( {
-					type: 'focus',
-				} )
+			await waitFor( () =>
+				expect( screen.getByText( 'tooltip text' ) ).toBeVisible()
 			);
 
 			// Wait for the tooltip element to be positioned (aligned with the button)
 			await waitFor( () =>
 				expect(
-					getWrappingPopoverElement( tooltip )
+					getWrappingPopoverElement(
+						screen.getByText( 'tooltip text' )
+					)
 				).toBePositionedPopover()
 			);
 		} );
 
-		it( 'should render children with additional tooltip when hovered', async () => {
-			const user = userEvent.setup( {
-				advanceTimers: jest.advanceTimersByTime,
-			} );
+		it( 'should render the tooltip when anchor is hovered', async () => {
```

Comment:
Good point!

Reply:
I updated to tooltip trigger, but what do you think of 'tooltip anchor'? 😂 In ariakit, they also use 'anchor' for their tooltip (`TooltipAnchor`) so I think it would be nice to keep that language.

Reply:
That also works!